### PR TITLE
refactor: move naive assertion to common role

### PR DESCRIPTION
**Pull request by zhan9san**
If this proposal is accepted, I'll update all other roles

### CODE REVIEW

**Review by SuperQ — APPROVED** [review]

**Review by gardar — CHANGES_REQUESTED** [review]
Thanks! Please go ahead with applying the change across all the roles

**Comment by zhan9san** [comment]
Done. It's ready for review

File: roles/smartctl_exporter/tasks/preflight.yml
Comment (gardar) [inline roles/smartctl_exporter/tasks/preflight.yml:1]: Don't forget to remove the naive assertion task here

**Review by gardar — CHANGES_REQUESTED** [review]

File: roles/smartctl_exporter/tasks/preflight.yml
Comment (zhan9san) [reply roles/smartctl_exporter/tasks/preflight.yml:1]: Good point. It's removed.

**Comment by zhan9san** [comment]
> I wonder if this assert should be added to the cadvisor role too.

I should list cadvisor in the description.

It is not added intentionally. Because it is not in standard format.

```shell
$ docker run --rm -it gcr.io/cadvisor/cadvisor --help | grep -E 'listen|port'
    	Only report docker containers in addition to root stats
  -listen_ip string
    	IP to listen on, defaults to all IPs
  -port int
    	port to listen (default 8080)
```

`node_exporter` and others is in this format, `--web.listen-address=:9100`.

**Review by gardar — APPROVED** [review]
LGTM

**Pull request merged**